Gemfile.lock

```diff
@@ -183,4 +183,4 @@ DEPENDENCIES
   uglifier
 
 BUNDLED WITH
-   2.0.2
+   2.1.3
```

Comment:
Can we separate dependency updates from this PR?

Reply:
sure, we just have different environment.